commit c0cce208e4a5c5b058916221764d7ebcb5c31e6b
Author: Samir Hussain <build@openswan.org>
Date:   Mon May 28 10:04:14 2018 -0400

    Update README to provide latest info on xl2tpd + Linux kernel 4.15+

diff --git a/README.xl2tpd b/README.xl2tpd
index b5b712d..30ee65a 100644
--- a/README.xl2tpd
+++ b/README.xl2tpd
@@ -28,9 +28,8 @@ Xl2tpd is based on the 0.69 L2TP by Jeff McAdams <jeffm@iglou.com>
 It was de-facto maintained by Jacco de Leeuw <jacco2@dds.nl> in 2002 and 2003.
 
 NOTE: In Linux kernel 4.15+ there is a kernel bug with ancillary IP_PKTINFO.
-      As such, for Linux kernel 4.15+ we recommend the community use the
-      1.3.12 branch as a temporary solution until the branch is merged into
-      the master branch.
+      As such, for Linux kernel 4.15+ we recommend the community use xl2tpd
+      1.3.12+
 
 Mailing Lists :
     https://lists.openswan.org/cgi-bin/mailman/listinfo/xl2tpd is home of the
